src/app/app.component.html

```diff
@@ -1,11 +1,52 @@
 <prx-auth [host]="authHost" [client]="authClient"></prx-auth>
-<prx-header>
-  <metrics-podcast-nav *ngIf="loggedIn$ | async"></metrics-podcast-nav>
-  <prx-navuser *ngIf="loggedIn$ | async" [userinfo]="userinfo$ | async">
-    <prx-spinner class="user-loading"></prx-spinner>
-    <prx-image *ngIf="userDoc$ | async" class="user-loaded" [imageDoc]="userDoc$ | async"></prx-image>
-  </prx-navuser>
-</prx-header>
+<div class="prx-header-bar">
+
+<header class="prx-header-bar-header">
+  <a class="navbar-brand d-inline-flex" href="/"><img width="152" height="30" alt="Dovetail from PRX" src="assets/images/dovetail_logo.svg"></a>
+</header>
+
+<nav class="prx-navbar-apps navbar-nav">
+  <h2 class="visually-hidden">Your Applications</h2>
+
+  <a class="nav-link" href="https://podcasts.dovetail.prx.org/">Podcasts</a>
+  <a class="nav-link active" href="/">Metrics</a>
+  <a class="nav-link" href="https://inventory.dovetail.prx.org">Inventory</a>
+</nav>
+
+<nav class="prx-dropdown navbar-nav" tabindex="0">
+  <h2 class="visually-hidden">Help Menu</h2>
+  <a class="prx-dropdown-toggle" href="#" title="Help">
+    <img alt="Help" src="assets/images/icon-help.svg">
+  </a>
+  <div class="prx-dropdown-menu">
+    <a class="prx-dropdown-item" href="https://help.prx.org">Help Docs</a>
+    <a class="prx-dropdown-item" href="https://status.prx.org">Status</a>
+    <a class="prx-dropdown-item" href="mailto:dovetail-feedback@prx.org">Send Feedback</a>
+    <a class="prx-dropdown-item" href="https://exchange.prx.org/terms-of-use">Terms</a>
+    <a class="prx-dropdown-item" href="https://dovetail.prx.org/attribution">Attribution</a>
+    <a class="prx-dropdown-item" href="https://exchange.prx.org/privacy-policy">Privacy</a>
+    <a class="prx-dropdown-item" href="https://prx.org/">PRX</a>
+  </div>
+</nav>
+
+<nav class="prx-user-menu prx-dropdown navbar-nav" tabindex="0">
+  <h2 class="visually-hidden">User Menu</h2>
+  <a class="user-menu-toggle" href="#" title="User Menu">
+    <img width="24" height="24" src="https://id.prx.org/pub/ac63a62099cf9b5ac48f69254c3e6bc9/user_images/52969/medium/P8140272_edited%20(1).jpg" class="prx-user-menu-avatar" alt="Brandon Hundt's avatar">
+  </a>
+  <div class="prx-dropdown-menu prx-user-menu-dropdown">
+    <div class="prx-user-menu-header">Brandon Hundt</div>
+    <a class="prx-user-menu-button" href="//id.prx.org/profile">Manage Account</a>
+    <a class="prx-user-menu-item" href="/sessions/refresh">Reload</a>
+    <a class="prx-user-menu-item" href="/sessions/logout">Logout</a>
+  </div>
+</nav>
+</div>
+
+<div class="prx-app-bar">
+  <metrics-nav-menu *ngIf="authorized$ | async"></metrics-nav-menu>
+</div>
```

Comment:
I wasn't able to test this, but the idea here is that if the user is logged in, they should get the podcast switcher in the app nav bar (like Feeder's podcast switcher). If I can figure out how to log in to the app, I can ensure the styles work for it.